### PR TITLE
updated the documenation to reflect the gitflow workflow and use of d…

### PR DESCRIPTION
**Pull request by balazagi**
update the git instructions to reflect the gitflow workflow and use of develop and feature branches. I have published the pages in here for a quick local view: https://www-nwp/~ssadri/catnip/

### CODE REVIEW

**Comment by nhsavage** [comment]
we have fixed the tests, so it should be possible to merge this now

**Comment by zmaalick** [comment]
> o it should be possible to merge this now

we can merge it as an administrator. But I would prefer to rebase and commit it. it will allow to run tests.

File: docsrc/instructions.rst [hunk 1]
```diff
@@ -92,50 +92,61 @@ Create a branch
...
 
 In Git, creating a branch is a way to start a separate line of development. This is usually done when a developer
-needs to fix a bug or add new features without affecting the current working code.
+needs to fix a bug or add new features without affecting the current working code. The branching method varies depending on the type of workflow  you
+have chosen for CATNIP we are using the Gitflow workflow. More details can be found here:
+https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
 
-To create a branch off the CATNIP master branch, make sure you are in the checked out directory of your repository,e.g.
+This workflow uses two branches to record the history of the project. A *master* branch to store the stable version of the project
+and a *develop* branch to serve for adding new features. CATNIP already has a *develop* so you only need to check it out, make sure you
```
Comment (nhsavage) [inline docsrc/instructions.rst:100]: I think that the rest of this section is redundant. as it duplicates what is below. 

I would just tweak this to say:
 _CATNIP already has a *develop* so you only need to check it out, and make your new branch from this (see next section)_

and delete the rest

**Review by nhsavage — CHANGES_REQUESTED** [review]

File: docsrc/instructions.rst (hunk 1 above)
Comment (balazagi) [reply docsrc/instructions.rst]: done the changes and pushed the branch

**Review by nhsavage — APPROVED** [review]

**Pull request merged**